nilearn/glm/first_level/first_level.py

```diff
@@ -361,6 +425,10 @@ def fit(self, run_imgs, events=None, confounds=None,
             Design matrices that will be used to fit the GLM. If given it
             takes precedence over events and confounds.
 
+        bins : int, optional
+            Maximum number of discrete bins for the AR coef histogram.
+            Default=100.
```

Comment:
this is not really an histogram for kmeans. It would call this adaptive quantification.

Reply:
Updated, thanks.

Reply:
not here.